fontir/src/ir.rs

```diff
@@ -387,11 +398,12 @@ fn adjust_offset(offset: f64, angle: f64) -> f64 {
     }
 }
 
-/// Map of values associated with a given global metric at various locations.
-pub type GlobalMetricValues = HashMap<NormalizedLocation, OrderedFloat<f64>>;
+/// Map of deltas associated with a given global metric across regions.
+pub type GlobalMetricDeltas = Vec<(VariationRegion, Vec<f64>)>;
```

Comment:
this is so slightly confusing... presumably the values at the default location are not deltas? And the values at other locations are relative to the default? This might need to be spelled out more explicitly.